definitions/variable/sdg-indicators/sdg.yaml

```diff
@@ -98,12 +98,12 @@
     sdg: 2
     unit: kcal/cap/day
     weight: Population
-- Price|Agriculture|Livestock|Index:
+- Price|Agriculture|Livestock [Index]:
     description: Weighted average price index of livestock
     sdg: 2
     unit: Index (2020 = 1)
     skip-region-aggregation: true
-#     weight: Agricultural Production|Non-Energy
+    shape: Price|Agriculture|Livestock|Index
```

Comment:
Maybe I missed this use, I don't know what the `shape` attribute is.

Reply:
"shape" is a previous project where this variable was already used, so this attribute is used similarly as "navigate" or "engage"...

Reply:
@khaeru suggested in #46 to use a dedicated attribute with sub-attributes for projects, but I suggest to merge this PR as is now and revisit/streamline later...

Reply:
Ah right of course, I read it not as the project. Maybe we should capitalize at some point to make it clearer `SHAPE, ENGAGE, ...`
But good for now.